src/chat/chatter.py

```diff
@@ -369,11 +373,22 @@ def set_color(self):
             color = pcolors.getUserColor(_id, name)
         self.setForeground(QtGui.QColor(color))
 
+    # temp sync api handling with request to fix viewAliases 12.2017
+    def return_decoded_json(self, link, method):
+        try:
+            with urllib.request.urlopen(link) as response:
+                return json.loads(response.read().decode())
+        except urllib.error.URLError as e:
+            logger.error("{} method error (link : {}) : {}".format(method, link, e.reason))
+            QtWidgets.QMessageBox.about(self.parent, "Aliases", "Error getting info, "
+                                                                "report the issue with logs on the faf forum")
+            return 'err'
```

Comment:
I think None would be better here, just so we don't confuse an arbitrary string with json.

Reply:
None might be more elegant, but in the 2 function using the return "if response == 'err':",
 that had to become "if response is None:"  and that wouldn't be true. (we don't want to lie, or?)

Reply:
In that case throw and catch an error, like ValueError or other standard python exception.

Reply:
that is not the point of the return and the error is thrown in the lines above it.

Reply:
but your the Boss .. have it your way.

Reply:
You can do it how you prefer, but please use some dedicated type instead of a string for reporting an error.

Reply:
here it's not a report. it's a very local flag.

Reply:
I don't understand what you mean. An error is a special condition, it should be reported with some dedicated value or type, not a magic string.

Reply:
You don't see the lines 382-383 (which do the reporting)?
And I already changed it .. so what is this about?
If you want something else, just tell me what you want there.

Reply:
Oh, I'm really sorry for that, my fault for not checking if it's outdated. You're absolutely right, it's fine right now.